src/common/com/intellij/plugins/haxe/haxelib/HaxelibUtil.java

```diff
@@ -134,6 +138,10 @@ public static VirtualFile getLibraryRoot(@NotNull Sdk sdk, @NotNull String libNa
       if (s.isEmpty()) continue;
       if (s.startsWith("-D")) continue;
       if (s.startsWith("-L")) continue;
+      if (s.matches("Error: .*")) {
+        logWarningEvent(HaxeBundle.message("haxelib.synchronization.title"), s);
```

Comment:
These type messages are going to the Event Log pane.  It communicates the error to the user much better than the idea.log.  :)

Reply:
BTW, I didn't add logging to the section above (lines 87-133) because that's the "optimized" path.  We don't consider issues at that point failures.